### PR TITLE
OSDOCS-5755

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-5755](https://issues.redhat.com//browse/OSDOCS-5755): Add URLs to firewall prerequisites

Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-5755

Link to docs preview:
https://59690--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-aws-prereqs.html#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @yuwang-RH! Can I please get a review of this PR? Thanks!

**Comment by jneczypor** [comment]
Hi @RH-pivancic! I thought you might like to take a look at this PR and make sure it includes everything you wanted. Thanks!

**Comment by jneczypor** [comment]
Hi @yuwang-RH! Can I please get a review of this? Thanks!

**Comment by jneczypor** [comment]
Hi @yunjiang29. Since you were tagged on the JIRA, I thought I should tag you here also. Can you please review this? Thanks!

**Comment by jneczypor** [comment]
Hi @arendej and @bchandra-ocp. May I please get a PM review of this PR? Thanks so much!

**Review by arendej — APPROVED** [review]
lgtm

**Review by skopacz1 — COMMENTED** [review]
Looks good to me! 

However, I'm new to the peer review squad, so please wait until @adellape gives this review a thumbs up or OK before proceeding with merge review. Thanks!

File: modules/osd-aws-privatelink-firewall-prerequisites.adoc
Comment (adellape) [inline modules/osd-aws-privatelink-firewall-prerequisites.adoc:66]: Init cap for "Operators", per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#operator).

**Review by adellape — COMMENTED** [review]
@skopacz1 @jneczypor Peer review follow-up complete 👍

**Pull request merged**
